### ISSUE TITLE
Unable to set additional parameters for google engine

### ISSUE THREAD

**Issue by unixfox** [searxng/searxng#2513]
If you try to set a proxy for btdigg for example it will send the request to a proxy:
```
- name: btdigg
    engine: btdigg
    shortcut: bt
    proxies:
       all://:
        - socks5h://127.0.0.1:9050
```

But for google it doesn't work:

```
- name: google
    engine: google
    shortcut: go
    proxies:
      all://:
        - socks5h://127.0.0.1:9050
```

You can also try any other parameter like `enable_http2: false`, HTTP2 will still be used.

I'm not sure if it affects other engines too.

@return42 do you know what is going on?

**Comment by return42** [comment]
About `enable_http2`: this option can't be set in the engine.  This option can (only) be set in the [`outgoing:` section](https://docs.searxng.org/admin/engines/settings.html#outgoing).

AFAIK its the same with the `proxies:` setting .. or did I miss something?

**Comment by return42** [comment]
Not sure if this issue is related to https://github.com/searxng/searxng/issues/762 ..

**Comment by unixfox** [comment]
Well if it can help you, it did work at the time of this random commit: https://github.com/searxng/searxng/commit/53a4a90d0c9bb69b995e3957bbe77aaba22370d2

I'll try to pinpoint the commit where it broke.

**Comment by return42** [comment]
I assume its broken (the google engine) since 2499899554 .. in this commit the import of the name `network`

https://github.com/searxng/searxng/blob/2e4a435134e0f677fbe24853dd81453a54770674/searx/engines/google.py#L26

has been moved from a function level to the module level .. 

This is what I pointed out in my comment of #762 .. but we have several other modules that do import this name .. I think its more a problem in general that we have conflicts on names in the module level of an engine module.

**Comment by return42** [comment]
@unixfox can you please test (review) https://github.com/searxng/searxng/pull/2517 to see if it fix the issue you reported here in this thread / thanks :+1:

**Comment by unixfox** [comment]
Solved in #2517
